src/test/java/com/connexta/multiintstore/MultiIntStoreApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package com.connexta.multiintstore;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class MultiIntStoreApplicationTests {
```

Comment:
Does this test succeed? What is it testing?

Reply:
It's a default test that ensures the app starts.